src/environment/mod.rs

```diff
@@ -156,7 +156,7 @@ impl<N: Network> Environment for MinerTrial<N> {
         "159.223.124.150:4132", "137.184.192.155:4132", "147.182.213.228:4132", "137.184.202.162:4132", "159.223.118.35:4132",
         "161.35.106.91:4132", "157.245.133.62:4132", "143.198.166.150:4132",
     ];
-    const MINIMUM_NUMBER_OF_PEERS: usize = 11;
-    const MAXIMUM_NUMBER_OF_PEERS: usize = 21;
+    const MINIMUM_NUMBER_OF_PEERS: usize = 21;
+    const MAXIMUM_NUMBER_OF_PEERS: usize = 31;
```

Comment:
Has someone verified that these new values do not result in blocking the miner from running?

Reply:
I'm not running a miner in the testnet, but the increased numbers for the client (which are greater) only led to ~5% avg CPU use; the change in peer counts can be done separately too